nhrpd/nhrp_vty.c

```diff
@@ -746,7 +746,7 @@ static void show_ip_nhrp_cache(struct nhrp_cache *c, void *pctx)
 				       nhrp_cache_type_str[c->cur.type]);
 		json_object_string_add(json, "protocol", buf[0]);
 		json_object_string_add(json, "nbma", buf[1]);
-		json_object_string_add(json, "claimed_nbma", buf[2]);
+		json_object_string_add(json, "claimedNbma", buf[2]);
```

Comment:
Please go through an appropriate deprecation cycle with this change.  Even if it is wrong

Reply:
I proposed to keep the old value.

Reply:
IMO, we should mark this as a deprecation anyway (claimed_nbma) because it's not compliant with our JSON requirements for the key names.

Reply:
@pguibert6WIND can you add deprecation for `claimed_nbma`? (CONFDATE)

Reply:
done